developers/dev-faq.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Development FAQ
+description: Commonly Asked Questions Regarding the Technical Aspects of the Ocean Stack
+---
+## Developement FAQ
+
+Have some questions about the Ocean Protocol tech stack?
+
+Hopefully, you'll find the answers here! If not then please don't hesitate to reach out to us on [discord](https://discord.gg/EdmenE7eTj) - there are no stupid questions!
+
+<details>
+<summary>The blockchain is public - does this mean that anyone can access my data?</summary>
+
+Your data is protected as it is stored encrypted on the chain. So no one is able to access your data via the blockchain without purchasing access (with the datatoken) through the smart contract. Ocean smart contracts encrypt the URL to the dataset before it is published on the blockchain. This means that only the encrypted URL will be queryable in the public blockchain. Ocean technology facilitates data access to the consumer via a proxy (Ocean Provider) and the unencrypted URL is never exposed.
+
+</details>
+
+<details>
+<summary>How are datatokens created, and how does the creator make money?</summary>
+
+[Datatokens](../developers/contracts/datatokens) represent access rights to a specific dataset or data service on the Ocean Protocol. Creators of datasets tokenize their data, creating these datatokens. Each datatoken is unique to a dataset.
```

Comment:
There really should be two separate questions: 
- How are datatokens created?
- How does the creator make money?

The current answer does a poor job on both. They need rewriting. Other parts of the docs give good answers. Best to give a 1-sentence summary on each, then point the reader to more detail.